src/ResourceManagement/DataFactory/DataFactoryManagement/Customizations/Models/Activities/HDInsightActivityBase.cs

```diff
@@ -32,6 +32,11 @@ public abstract class HDInsightActivityBase : ActivityTypeProperties
         /// </summary>
         public IList<string> Arguments { get; set; }
 
+        /// <summary>
+        /// The <see cref="HDInsightActivityDebugInfoOption"/> settings to use.
+        /// </summary>
+        public string GetDebugInfo { get; set; }
```

Comment:
Because it markes if we should get debug information from HDINSIGHT cluster or not